updater/src/Updater/SyncLabels/IssueLabel.purs

```diff
@@ -35,27 +35,27 @@ issueLabelCodec =
 labels :: Array IssueLabel
 labels =
   [ { name: "breaking change"
-    , description: "Change will require a major version bump"
+    , description: "A change that requires a major version bump"
     , color: "e99695"
     }
   , { name: "bug"
-    , description: "Something isn't working"
+    , description: "A legitimate bug in the library that ought to be fixed"
     , color: "d73a4a"
     }
   , { name: "document me"
     , description: "Improvements or additions to documentation"
     , color: "0075ca"
     }
   , { name: "enhancement"
-    , description: "New feature or request"
+    , description: "An addition to the library"
     , color: "a6e1ea"
     }
   , { name: "good first issue"
     , description: "Good for newcomers"
     , color: "7057ff"
     }
   , { name: "help wanted"
-    , description: "Extra attention is needed"
+    , description: "Maintainers would like a PR that solves this issue"
```

Comment:
I feel like this is implied for all issues.
Would it make more sense to have a "help not wanted - maintainer will sort this out themselves" tag? Seems like issue assignment takes care of that use case.

Reply:
I think "help wanted" is usually indicative of something that the maintainers would like to happen, but don't have the bandwidth to tackle.

Reply:
It could certainly be implied that any open issue is one that the maintainers want help on, but sometimes open issues are there primarily for organization purposes, and not necessarily something that would be good for a new contributor to pick up.

So I think explicitly saying "we'd like help with this" is a good signal (and one that is fairly standard across GitHub).

Reply:
That makes sense. Wasn't considering the cases where library users open issues for stuff that either require no code changes, or that the maintainer doesn't want changed.

Reply:
I agree that the 'help wanted' sends a more explicit "we do want help with this" signal, which is why it's included.